### ISSUE TITLE
nValue issue?

### ISSUE THREAD

**Issue by perhusaas** [flopp999/Tibber-Domoticz#43]
This plugin runs ok and gets values, but its throwing these log errors?

```
2021-11-28 16:37:39.150 Error: Python Plugin System: (Tibber) 'onMessage' failed 'NameError':'name 'nValue' is not defined'.
2021-11-28 16:37:39.150 Error: Python Plugin System: (Tibber) ----> Line 509 in '/opt/domoticz/userdata/plugins/Tibber-Domoticz/plugin.py', function onMessage
2021-11-28 16:37:39.150 Error: Python Plugin System: (Tibber) ----> Line 276 in '/opt/domoticz/userdata/plugins/Tibber-Domoticz/plugin.py', function onMessage
2021-11-28 16:37:39.150 Error: Python Plugin System: (Tibber) ----> Line 493 in '/opt/domoticz/userdata/plugins/Tibber-Domoticz/plugin.py', function UpdateDevice
```

**Comment by flopp999** [comment]
Hi what version do you run?

Is these errors appear when you start the plugin or random?

**Comment by perhusaas** [comment]
I used the main branch, fetched a few days ago? Its coming in random, not that often any more it seems. 

I also have a Tibber Pulse, but no "current consumption", just prices?

**Comment by perhusaas** [comment]
Very good! I just tried it now, and it works for pricing, but still no watts. Theres also some errors:
`2022-01-05 21:32:50.316 Error: Python Plugin System: (Tibber) Missing packages
2022-01-05 21:32:50.316 Error: Python Plugin System: (Tibber) [ModuleNotFoundError("No module named 'websockets'")]
2022-01-05 21:33:01.968 Error: Python Plugin System: (Tibber) 'onHeartbeat' failed 'NameError'.
2022-01-05 21:33:01.968 Error: Python Plugin System: (Tibber) ----> Line 533 in '/opt/domoticz/userdata/plugins/Tibber-Domoticz/plugin.py', function onHeartbeat
2022-01-05 21:33:01.968 Error: Python Plugin System: (Tibber) ----> Line 336 in '/opt/domoticz/userdata/plugins/Tibber-Domoticz/plugin.py', function onHeartbeat
2022-01-05 21:33:01.968 Error: Python Plugin System: (Tibber) ----> Line 50 in '/usr/lib/python3.7/asyncio/runners.py', function run
2022-01-05 21:33:01.968 Error: Python Plugin System: (Tibber) ----> Line 584 in '/usr/lib/python3.7/asyncio/base_events.py', function run_until_complete
2022-01-05 21:33:01.968 Error: Python Plugin System: (Tibber) ----> Line 320 in '/opt/domoticz/userdata/plugins/Tibber-Domoticz/plugin.py', function LivePower`

The two first lines are just in startup, the rest of them repeats at a certain interval.
I run domoticz(build 12917) in a docker, python 3.7. I did run the "requirements.txt" command?

I tried to add "websockets" manually, but that was a problem for my Zigate plugin for some reason..
